cmd/loadtest/loadtest.go

```diff
@@ -310,6 +310,10 @@ func initNonce(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Client) err
 
 func completeLoadTest(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Client) error {
 	log.Debug().Uint64("startNonce", startNonce).Uint64("lastNonce", currentNonce).Msg("Finished main load test loop")
+	if *inputLoadTestParams.SendOnly {
+		log.Info().Msgf("Send only mode enabled: finished sending %d transactions.", currentNonce-startNonce)
```

Comment:
Super minor comment. With structured logs like this, I usually avoid using `Msgf` and `fmt.Sprintf`. Something like this would be an alternative to avoid string formatting.
```suggestion
		log.Info().Uint64("transactionsSent", currentNonce-startNonce).Msg("SendOnly mode enabled - skipping wait period and summarization")
```

Reply:
Updated!